sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/query.sql

```diff
@@ -200,6 +207,10 @@ SELECT
       _previous.metadata.min_first_session_ping_run_date,
       _current.metadata.min_first_session_ping_run_date
     ) AS min_first_session_ping_run_date,
+    COALESCE(
+      _previous.metadata.min_metrics_ping_submission_date,
+      _current.metadata.min_metrics_ping_submission_date
+    ) AS min_metrics_ping_submission_date,
```

Comment:
This should take the min:
```suggestion
    LEAST(
      _previous.metadata.min_metrics_ping_submission_date,
      _current.metadata.min_metrics_ping_submission_date
    ) AS min_metrics_ping_submission_date,
```

Reply:
The problem with LEAST is that it returns null as minimun if any value is null. We'd need a CASE,

Reply:
Yeah, then we'll have to handle nulls. But we definitely want the minimum